src/components/time/ReadableTime.stories.js

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import ReadableTime from './ReadableTime';
+import notes from './ReadableTime.stories.md';
+
+export const relativeTimestamps = () => {
+    const msInADay = 24 * 60 * 60 * 1000;
+    const oneHourInMs = 60 * 60 * 1000;
```

Comment:
maybe we could make these constants scoped to the file since it's used in the other examples?